app/src/main/res/layout-sw600dp-port/question_player_fragment.xml

```diff
@@ -97,6 +97,38 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
+      <FrameLayout
```

Comment:
These changes are not correct as per this comment: https://github.com/oppia/oppia-android/pull/2540#pullrequestreview-578738098